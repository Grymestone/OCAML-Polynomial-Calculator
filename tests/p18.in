(28a^2 - 1a^2) + (32a^3 - 27a^8)^2 - (32a^1 + 1a^3)
