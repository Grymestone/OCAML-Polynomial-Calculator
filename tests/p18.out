((((28*(a)^2)-(1*(a)^2))+(((32*(a)^3)-(27*(a)^8)))^2)-((32*(a)^1)+(1*(a)^3)))
-1728x^11 + 729x^16 + 1024x^6 + -1x^3 + 27x^2 + -32x^1